source/_header.erb

```diff
@@ -1,9 +1,12 @@
 <div id="header">
   <div id="site-title">
-<span class="line"><%= link_to 'LRUG', '/' %></span> : <span class="line">london</span> <span class="line ruby"><%= link_to 'ruby', 'http://www.ruby-lang.org/en/' %></span> <span class="line">user</span> <span class="line">group</span>
```

Comment:
I wasn't sure why `class="line"` was used here. In the CSS they we're `display: inline`. I think it's a relic of a previous design maybe.

Reply:
Oh, that _is_ weird.  I suspect you're right and they're from a previous design as they're not necessary at all with the current design.